### PR TITLE
Disable hide/unhide options for filesystem hidden folders

### PR DESCRIPTION
**Pull request by esensar**

### CODE REVIEW

**Comment by esensar** [comment]
This disables hide/unhide options for folders that are hidden in filesystem (due to having name starting with `.`), because the way hiding works in this application will not change that fact, only renaming would. Not sure if we need to add some additional information to make it clearer to the users why that option is not present.

**Comment by tibbi** [comment]
how are filesystems specified there? Wont those menu items be always visible?

**Comment by esensar** [comment]
This PR will disable both hide and unhide options for folders that are hidden by having their name start with a dot, regardless of whether they have `.nomedia` file.

**Comment by tibbi** [comment]
Im not sure if this is the correct way of doing it :/ Shouldnt we check if the folder starts with /android/data or so?

**Comment by esensar** [comment]
Hmm, I am not sure if we are seeing the same problem then. The only folders for which I have seen confusing behavior are the ones whose name starts with a dot, meaning they are hidden (an example of such folder is WhatsApp Statuses - `.Statuses`). If you have example of application that behave like that without having `.` in their name, let me know so I can test that out too.

**Comment by tibbi** [comment]
Im afraid this will make people write bad reviews or emails about Un/hide button missing, it needs some explanation. Having the hide button sometimes visible and sometimes not just isnt user friendly enough. How about changing it a bit and at pressing Unhide showing a basic confirmation dialog with text `If a folder or one of its parent folders has a dot before its name, it is hidden and cannot be unhidden like this. You have to remove the dot by renaming it.`. After the user confirms, just close the dialog, do not do anything. And do it at Unhide only, do not affect hiding functionality with this PR. Dont show anything, dont change when is the button visible.

File: app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt [hunk 1]
```diff
@@ -305,6 +305,11 @@ class DirectoryAdapter(
                 }
             }
         } else {
+            if (selectedPaths.any { File(it).isHidden }) {
```
Comment (tibbi) [inline app/src/main/kotlin/com/simplemobiletools/gallery/pro/adapters/DirectoryAdapter.kt:308]: we should check all parent folders, not just this one

**Comment by tibbi** [comment]
alright, should be good now thanks

**Pull request merged**